src/sqon/searchSqon.ts

```diff
@@ -44,7 +44,7 @@ export const searchSqon = async (
         throw new Error(get(results, 'errors', undefined));
```

Comment:
idKey is an env var. Currently set to `fhir_id`. Variants documents have no such value. The change supposes that `idField` is the kind of id to be saved (already the case) AND the property to target to get back the results.